package.json

```diff
@@ -89,12 +49,13 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/chuck-adaptive/ui-kit-2022.git"
+    "url": "git+https://github.com/AdaptiveConsulting/ui-kit-2022.git"
   },
   "bugs": {
-    "url": "https://github.com/chuck-adaptive/ui-kit-2022/issues"
+    "url": "https://github.com/AdaptiveConsulting/ui-kit-2022/issues"
   },
   "readme": "ERROR: No README data found!",
-  "homepage": "https://github.com/chuck-adaptive/ui-kit-2022#readme",
-  "_id": "ui-kit-2022@0.0.0"
+  "homepage": "https://github.com/AdaptiveConsulting/ui-kit-2022#readme",
+  "_id": "ui-kit-2022@0.0.0",
+  "packageManager": "yarn@3.2.4"
```

Comment:
Interesting - was yarn version 3 required to get the mono-repo tooling working?

Do you have do a global install for yarn w/ this version?   (If so, might be nice to add that as a step in the `README.md`

Reply:
Yeah, the build commands require yarn2+. The newer yarn versions have better tooling when working with monorepos.  I'll add info about how to configure the local dev environment in the readme, as well as how to run the builds and start developing.